presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -164,4 +187,31 @@ private String indentString(int indent)
     {
         return Strings.repeat("    ", indent);
     }
+
+    private static class AnyQualifiedNameReference extends QualifiedNameReference
```

Comment:
It sounds like a hack to extend `QualifiedNameReference`. Maybe you can just extend from `Expression` and then you could name this as `AnyReference`